plugin-base/src/main/java/io/everytrade/server/plugin/impl/everytrade/BlockchainLtcConnector.java

```diff
@@ -0,0 +1,94 @@
+package io.everytrade.server.plugin.impl.everytrade;
+
+import io.everytrade.server.model.SupportedExchange;
+import io.everytrade.server.plugin.api.IPlugin;
+import io.everytrade.server.plugin.api.connector.ConnectorDescriptor;
+import io.everytrade.server.plugin.api.connector.ConnectorParameterDescriptor;
+import io.everytrade.server.plugin.api.connector.ConnectorParameterType;
+import io.everytrade.server.plugin.api.connector.DownloadResult;
+import io.everytrade.server.plugin.api.connector.IConnector;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+public class BlockchainLtcConnector implements IConnector {
+    private static final String ID = EveryTradePlugin.ID + IPlugin.PLUGIN_PATH_SEPARATOR + "blockchainLtcApiConnector";
+
+    private static final ConnectorParameterDescriptor PARAMETER_ADDRESS =
+        new ConnectorParameterDescriptor(
+            "ltubKey",
```

Comment:
Change to:

  "source",
            ConnectorParameterType.STRING,
            "Wallet address / ltub key",
            ""